src/style.css

```diff
@@ -0,0 +1,131 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-bar-big {
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+  height: 70px;
+  position: relative;
+}
+
+.search-bar-small {
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+  height: 42px;
+  position: relative;
+}
+
+.big-board {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  font-family: inherit;
+  outline: none;
+  padding-left: 62px;
+}
+
+.icon-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.small-board {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  font-family: inherit;
+  outline: none;
+  padding-left: 33px;
+
+}
+
+.icon-small {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.big-board::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  font-weight: 300;
+}
+
+.small-board::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  padding-top: 12px;
+  padding-bottom: 10px;
+  font-weight: 300;
+}
+
+.big-board:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.big-board:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-sizing: border-box;
+  font-weight: bold;
+  font-size: 16px;
```

Comment:
Try to set all styles for input.
 For focus just change:
 background, box-shadow, border-radius, font-weight,